news/2025/2025-01-31-osumania-7k-world-cup-2025-round-of-16-recap.md

```diff
@@ -0,0 +1,233 @@
+
+Our second highlight of the week was [Supa7onyz feat. 5-let - Les Cadences Magiques](https://osu.ppy.sh/beatmapsets/2310776#mania/4943408) (HB2) by [_Yiiiii](https://osu.ppy.sh/users/6066359). How convenient, as both of this week's spotlighted picks were made by the same mapper! Silly trivia aside, HB2 is the exact opposite of RC5. With the second lowest pick rate of the round, outside of the tiebreaker, HB2 acted as the de facto wildcard pick for the pool. This was the definition of a high-risk, high-reward pick for those who were able to use it wisely to force their opponents to play a rather tricky map.
+
+Those who have been keeping up with the next round's pool know what's coming on the next recap! For the time being, the Round of 16 didn't quite yet have the chance to showcase a brand-new track. That's bound to change in the rounds to come, however, so stay tuned for the Quarterfinals and beyond!
+
+## Round of 16 summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| Thailand ::{ flag=TH }:: | 0 | **5** | ::{ flag=EE }:: **Estonia** |
+| Lithuania ::{ flag=LT }:: | 0 | **5** | ::{ flag=TW }:: **Taiwan** |
+| Mexico ::{ flag=MX }:: | 1 | **5** | ::{ flag=MO }:: **Macau** |
+| United Kingdom ::{ flag=GB }:: | 4 | **5** | ::{ flag=DK }:: **Denmark** |
+| **Netherlands** ::{ flag=NL }:: | **5** | 4 | ::{ flag=FI }:: Finland |
+| Peru ::{ flag=PE }:: | 3 | **5** | ::{ flag=MY }:: **Malaysia** |
+| Czechia ::{ flag=CZ }:: | 1 | **5** | ::{ flag=SE }:: **Sweden** |
+| **United States** ::{ flag=US }:: | **5** | 0 | ::{ flag=CL }:: Chile |
+| **Philippines** ::{ flag=PH }:: | **5** | 2 | ::{ flag=NZ }:: New Zealand |
+| **Japan** ::{ flag=JP }:: | **5** | 0 | ::{ flag=DE }:: Germany |
+| **Singapore** ::{ flag=SG }:: | **5** | 4 | ::{ flag=PL }:: Poland |
+| **South Korea** ::{ flag=KR }:: | **5** | 0 | ::{ flag=ES }:: Spain |
+| **China** ::{ flag=CN }:: | **5** | 0 | ::{ flag=FR }:: France |
+| **Hong Kong** ::{ flag=HK }:: | **5** | 1 | ::{ flag=RU }:: Russian Federation |
+| **Indonesia** ::{ flag=ID }:: | **5** | 1 | ::{ flag=VE }:: Venezuela |
+| Norway ::{ flag=NO }:: | 2 | **5** | ::{ flag=CA }:: **Canada** |
+
+As our teams start getting eliminated for the first time in the group stage, things have mostly remained quite chill. While there have been a fair share of interesting matches — particularly from those teams with closely matched seedings — matches continued being fairly lopsided during these early rounds.
+
+Unsurprisingly, the topmost teams have been exceeding our expectations with their breathtaking performances all throughout the pool. On that note, the top 7 seeded teams remain uncontested, making it seem like the increase in the pool's difficulty is negligible!
+
+But results by themselves do not tell the whole story. All of our 32 teams have given it their all this weekend. Regardless of who remains a victor, their efforts have not gone to waste in the end.
```

Comment:
i can understand when you talk about your expectations but im still unsure about "our" teams
```suggestion
But results by themselves do not tell the whole story. All of the 32 teams have given it their all this weekend. Regardless of who remains a victor, their efforts have not gone to waste in the end.
```